core/src/main/java/org/web3j/protocol/core/RemoteCall.java

```diff
@@ -44,16 +44,7 @@ public T send() throws Exception {
      *
      * @return an observable
      */
-    public Observable<T> observable() {
-        return Observable.create(
-                subscriber -> {
-                    try {
-                        subscriber.onNext(send());
-                        subscriber.onCompleted();
-                    } catch (Exception e) {
-                        subscriber.onError(e);
-                    }
-                }
-        );
+    public Flowable<T> observable() {
```

Comment:
I presume this syntax is accurate for exception handling and completion?

Reply:
Correct - it has been collapsed into one operation.
Ref: https://artemzin.com/blog/rxjava-defer-execution-of-function-via-fromcallable/